todo_app/view_model.py

```diff
@@ -0,0 +1,23 @@
+#@classmethod
+class ViewModel:
+    def __init__(self, items):
+        self._items = items
+
+    @property
+    def items(self):
+        return self._items
+    
+    @property
+    def done_items(self):
+        done_items = [item for item in self._items if item['status']=="done"]         
```

Comment:
As above, this might need to shift just slightly to work with the Item class:
```suggestion
        done_items = [item for item in self._items if item.status == "done"]         
```

This might be why it didn't seem to work on your homepage when you tried it